CHANGELOG.md

```diff
@@ -2,8 +2,14 @@
 
 ## next release
 ### Breaking
+
+* remove AutoRuleCorpusTester
+
 ### Features
 ### Improvements
+
+* remove AutoRuleCorpusTester
+
```

Comment:
Are both entries neccessary?

Reply:
Sure... Because the one Signals the kind of the change and the other signals if the change is a breaking change